docs/variables.md

```diff
@@ -184,6 +184,18 @@ Only available for `successCondition`
 | `response.body` | Response body (`string`) |
 | `response.headers` | Response headers (`map[string][]string`) |
 
+### CronWorkflow Templates
+
+> v3.6 and after, note that the the variables here can be used with the full power of expr-lang expression.
+
+| Variable | Description|
+|----------|------------|
+| `cronworkflows.name` | Name of the CronWorkflow method (`string`) |
+| `cronworkflows.numActive` | Number of running CronWorkflows (`int`) |
+| `cronworkflows.lastScheduledTimeNull` | When `cronworkflows.lastScheduledTime` is a sane value this is set to `false` else `true` (`bool`) |
```

Comment:
Can't we just make `lastScheduledTime` as `nil`?

This should also be simplified and avoid [assumption words](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-words-that-assume-a-specific-level-of-understanding)

I would also recommend the substantially easier to understand and more conventional `hasLastScheduledTime` if truly necessary 

```suggestion
| `cronworkflows.hasLastScheduledTime` | `true` if `cronworkflows.lastScheduledTime` exists else `false` (`bool`) |
```

Reply:
> Can't we just make lastScheduledTime as nil in this case?

No we cannot really do this, but changed to `hasLastScheduledTime`

Reply:
Can you elaborate on why not?

Reply:
We discussed this in [last week's Contributor Meeting](https://docs.google.com/document/d/1tznN5LB-KrNkC6dmeIzpKfuyvZWgCJpRuS-E84zseC8/edit#heading=h.jh0wp5re8e6), and it seemed like this should be doable to set as `nil` as `expr` can handle that and there are existing Argo variables that can be `nil` or a zero value in certain scenarios.

My suggestion was to therefore leave `nil`, but in docs and examples, always show how to handle this scenario